.env

```diff
@@ -1,4 +1,4 @@
-NODE_ENV=staging
-API_URL=https://api-staging-wire.andela.com
+NODE_ENV=development
+API_URL=http://localhost:3000/api
```

Comment:
@jimmykimani Take over this task. Revert the changes to the .env file and ensure that's it working as expected. It was meant to add a filter by assignees

Reply:
Noted